static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -122,7 +122,9 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
 
         <Section icon={<IconLink size="xs" />} title={t('Url')}>
           <ErrorBoundary mini>
-            <TextCopyInput size="sm">{url?.value ?? t('URL not found')}</TextCopyInput>
+            <TextCopyInput size="sm">
+              {eventData?.tags ? (url ? url.value : t('URL not found')) : ''}
```

Comment:
this checks if the eventData tags are loaded yet. if they've loaded, and there's no URL, then we can safely say no URL found... otherwise, tags are still loading so display empty string